traces

```diff
@@ -78,27 +82,61 @@ const REGEX = '/=([[:digit:]]+)>; rel="last"/';
                     $config['keepExcludedUsers'] ?? false,
                     $config['extractEmailDomain'] ?? false,
                     $config['fieldsWhitelist'] ? array_flip($config['fieldsWhitelist']) : [],
+                    $config['excludeRepositories'] ?? [],
                 ];
             })(
                 $configFile
             );
 
             $authHeaders = ['auth' => [$user, $password]];
-            $requestUri = str_replace('{repository}', $repository, GITHUB_CONTRIBUTORS_POINT);
 
             $client = new Client(['timeout' => $timeout,]);
 
-            $io->text('Loading contributors...');
-            $response = $client->get($requestUri);
+            $io->text('Loading repositories...');
+            if (substr_count($organization, '/') > 0) {
```

Comment:
You shouldn't use `$organization` for repository. The tool needs both parameters wdyt?

Reply:
Not for the moment. 

I want to keep the initial behavior until we decide to move to organization ;)

Reply:
At this moment, i will make a new PR ;)